rest_framework_csv/tests.py

```diff
@@ -247,7 +243,7 @@ def test_parse_file_with_only_carriage_returns(self):
 
         parser = CSVParser()
 
-        with open(CSVFILE, 'rbU' if sys.version_info <= (3, 10) else 'rb') as csv_file:
+        with open(CSVFILE, 'rb') as csv_file:
```

Comment:
The `U` flag triggers a `DeprecationWarning` on Python 3.8 still, and is not necessary.